src/tilesource.js

```diff
@@ -191,7 +196,11 @@ $.TileSource = function( width, height, tileSize, tileOverlap, minLevel, maxLeve
         this.aspectRatio = ( options.width && options.height ) ?
             (  options.width / options.height ) : 1;
         this.dimensions  = new $.Point( options.width, options.height );
-        this.tileSize    = options.tileSize ? options.tileSize : 0;
+        
+        this.tileSize = options.tileSize ? options.tileSize : 0;
```

Comment:
I think it's time to finally deprecate the tileSize property of TileSource (not the tileSize option for the TileSource constructor, which should stay). Here set `this.tileWidth` and `this.tileHeight` to `options.tileSize` if present, otherwise use the tileWidth and tileHeight options (or 0 if they are absent). 

Since we're deprecating the tileSize property, please remove the documentation for it at: 

https://github.com/openseadragon/openseadragon/blob/master/src/tilesource.js#L140-146

Speaking of such things, I'd like to move toward our "private" properties being prefixed with an underscore, so please change `this.tileWidth` to `this._tileWidth`. This was we give a hint to people to always use the accessor.